.github/workflows/build-and-test-orchard-core.yml

```diff
@@ -56,6 +56,20 @@ on:
           (https://github.community/t/how-to-clear-cache-in-github-actions/129038/5) until the associated issue
           (https://github.com/actions/cache/issues/2) is resolved and the feature is available not just via the RESTful
           API.
+      set-up-sql-server:
+        required: false
+        type: string
+        default: "true"
+        description: >
+          If set to "true", SQL Server Express will be installed and started. This is necessary if the app's UI tests
+          connect to an SQL Server database.
```

Comment:
Nitpicker's corner: I would prefer "**a SQL Server**", because it's very common to pronounce SQL Server as "sequel server".

Here's some very interesting historical background: https://learnsql.com/blog/sql-or-sequel/

For me the essence is, that MS SQL Server is pronounced "Em Es Sequel Server" (like Bill Gates does) while the language SQL is pronounced "ES Que EL".

Reply:
Fine :D.